de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/InitializationCallbacks.java

```diff
@@ -39,7 +41,7 @@
         private final DesktopPresenterImpl presenter;
         private final UserSessionServiceFacade userSessionService;
         private final UserPreferencesCallback userPreferencesCallback;
-        Logger LOG = Logger.getLogger(BootstrapCallback.class.getName());
+        Logger LOG = Logger.getLogger("BootstrapCallback");
```

Comment:
The more I think about it, I should just prioritize that ticket.  So yeah, I'll change it back 😄